compiler/rustc_mir/src/borrow_check/diagnostics/region_name.rs

```diff
@@ -6,8 +6,10 @@ use rustc_hir::def::{DefKind, Res};
 use rustc_middle::ty::print::RegionHighlightMode;
 use rustc_middle::ty::subst::{GenericArgKind, SubstsRef};
 use rustc_middle::ty::{self, RegionVid, Ty};
-use rustc_span::symbol::kw;
-use rustc_span::{symbol::Symbol, Span, DUMMY_SP};
+use rustc_span::{
+    symbol::{kw, sym, Ident, Symbol},
+    Span, DUMMY_SP,
+};
```

Comment:
Please don't split use statements to multiple lines, instead expand this to two use statements (as before).